docs/user-guide/limitations.md

```diff
@@ -1,15 +1,45 @@
 # Sysbox User Guide: Functional Limitations
 
 This document describes functional restrictions and limitations of Sysbox and
-system containers.
+the containers created by it (i.e., system containers).
 
 ## Contents
 
+-   [Sysbox Container Limitations](#sysbox-container-limitations)
 -   [Docker Restrictions](#docker-restrictions)
 -   [Kubernetes Restrictions](#kubernetes-restrictions)
--   [System Container Limitations](#system-container-limitations)
 -   [Sysbox Functional Limitations](#sysbox-functional-limitations)
 
+## Sysbox Container Limitations
+
+Sysbox enables containers to run applications or system software such as
+systemd, Docker, Kubernetes, K3s, etc., seamlessly & securely (e.g., no
+privileged containers, no complex setups).
+
+While our goal is for Sysbox containers to run **any software that runs on
+bare-metal or VMs**, this is still a work-in-progress. 
+
+Thus, there are some limitations at this time. The table below describes these.
+
+| Limitation    | Description       | Affected Software | Planned Fix |
+| --------      | --------------- | ----- | :--: |
+| mknod         | Fails with "operation not permitted". | Software that creates devices such as /dev/tun, /dev/tap, /dev/fuse, etc. | Yes |
+| binfmt-misc   | Fails with "permission denied". | Software that uses `/proc/sys/fs/binfmt_misc` inside the container (e.g., buildx+QEMU for multi-arch builds). | WIP |
+| unshare -U --mount-proc | Fails with "invalid argument". | Software that uses the Linux user-namespace (e.g., Docker + userns-remap). Note that the Sysbox container is rootless already, so this implies nesting Linux user-namespaces. | Yes |
+| Host device access | Host devices exposed to the container (e.g., docker run --devices ...) show up with "nobody:nogroup" ownership. Thus, access to them may fail with "permission denied". | Software that needs access to hardware accelerators. | Yes | 
+| rpc-pipefs    | Mounting rpc-pipefs fails with "permission denied". | Running an NFS server inside the Sysbox container. | Yes |
+| host sockets  | Host sockets exposed to the container (e.g., docker run -v /host/socket:/mount/point ...) don't always work as Sysbox mounts shiftfs on them. | Software that requires host socket access (e.g, VNC). | Yes |
+| insmod        | Fails with "operation not permitted". | Can't load kernel modules from inside containers. | TBD |
+
+
+**NOTES:** 
+
+* Planned Fix "WIP" means the fix is being worked-on right now. "TBD" means a
+  decision is yet to be made.
+
+* If you find other software that fails inside the Sysbox container, please open
+  a GitHub issue so we can add it to the list and work on a fix.
+
```

Comment:
I would summarize the text below by dumping it on this new table, especially since there's a clear overlap between (e.g.) docker's `--network=host` and k8s `hostNetwork: true`, and similar attributes -- we could create one single row for both docker and k8s case for each attribute. The same with the other attributes.

Reply:
I think it's better to keep the limitations of "sysbox container workloads" separate from the limitations of "sysbox integrations (with Docker, K8s, etc)". Otherwise it will be confusing I think.

Reply:
Yes, you're right, let's leave it like this.

Reply:
But I replace the rest of the text in the doc with tables, much clearer now I think.